v3/docs/07-advanced/h-ss58/index.mdx

```diff
@@ -80,25 +180,20 @@ const isValid = isValidSubstrateAddress()
 console.log(isValid)
 ```
 
-### Community Built
+### Ecosystem tools
 
 Below you will find a list of other Substrate API tools built in various languages by our community.
 They each validate addresses in their own way, and can act as a starting point.
 
 - [Python Substrate Interface](https://polkascan.github.io/py-substrate-interface/#substrateinterface.Keypair) - developed by Polkascan.
-
 - [Go Substrate RPC client](https://github.com/centrifuge/go-substrate-rpc-client) - developed by Centrifuge.
-
 - [C++ Polkadot Substrate API](https://github.com/usetech-llc/polkadot_api_cpp) - developed by Usetech.
-
 - [.Net API](https://github.com/usetech-llc/polkadot_api_dotnet) - developed by Usetech.
-
 - [Go implemented utilities](https://github.com/itering/subscan-essentials) - developed by Subscan.
-
 - [SS58 Transform](https://polkadot.subscan.io/tools/ss58_transform) - to manually verify an address (sanity check).
 
-### Learn more
-
+## Learn more
+- [`Ss58Codec` in Substrate](https://paritytech.github.io/substrate/master/sp_core/crypto/trait.Ss58Codec.html).
```

Comment:
```suggestion
- [`Ss58Codec` in Substrate](https://paritytech.github.io/substrate/master/sp_core/crypto/trait.Ss58Codec.html)
```
Refer to [style guide](https://docs.substrate.io/v3/contribute/writing/#bullets) on using fullstops in bulleted lists.